controller/workitem_blackbox_test.go

```diff
@@ -41,6 +41,11 @@ import (
 	"github.com/stretchr/testify/suite"
```

Comment:
It's not about whether or not it executes the same code. A test should capture the expected behavior in different scenarios. 

Maybe I missed it, but I couldn't see a Test that shows that ID = "0" is handled in any special way at all. 

And why is it ID = "0" which sounds like a fairly magic string as oppose to ID = nil?